src/bwtraceboundary.jl

```diff
@@ -54,7 +54,7 @@ julia> boundary[3]
 ```
 """
 function bwtraceboundary(
-    image::Union{Matrix{Int64},Matrix{Float64},T};
+    image::Union{Matrix{UInt8},Matrix{Int64},Matrix{Float64},T};
     P0::Union{Tuple{Int,Int},CartesianIndex{2},Nothing}=nothing,
     closed::Bool=true,
 ) where {T<:AbstractArray{Bool,2}}
```

Comment:
Because both `UInt8` and `Int64` are subtypes of `Integer`, the function signature could be simplified to something like
```julia
function foo(
    image::AbstractArray{T,2};
    P0::Union{Tuple{Integer,Integer},CartesianIndex{2},Nothing}=nothing,
    closed::Bool=true,
) where {T<:Union{Integer,AbstractFloat,Bool}}
```

Reply:
I like this suggestion, but I think it might be best in a separate PR. I'll make an issue. #460